openproblems/tasks/chromatin_potential/methods/beta.py

```diff
@@ -69,33 +87,38 @@ def _get_annotation(adata, retries=3):
     )
 
 
-def _atac_genes_score(adata, top_genes=500, threshold=1):
+def _atac_genes_score(adata, top_genes=2000, threshold=1, method="beta"):
     """Calculate gene scores and insert into .obsm."""
     import pybedtools
 
     # get annotation for TSS
     _get_annotation(adata)
     # basic quality control
-    if adata.shape[0] >= 600:
+    if adata.shape[0] >= 10000:
         sc.pp.filter_cells(adata, min_genes=200)
         sc.pp.filter_genes(adata, min_cells=5)
 
     adata = adata[:, ~pd.isnull(adata.var.loc[:, "chr"])].copy()
-    adata.var["mt"] = adata.var.gene_short_name.str.lower().str.startswith(
-        "mt-"
-    )  # annotate the group of mitochondrial genes as 'mt'
+    if adata.uns["species"] == "mus_musculus":
+        adata.var["mt"] = adata.var.gene_short_name.str.lower().str.startswith(
+            "mt-"
+        )  # annotate the group of mitochondrial genes as 'mt'
+    else:
+        adata.var["mt"] = adata.var.gene_short_name.str.lower().str.startswith(
+            "MT-"
+        )  # annotate the group of mitochondrial genes as 'mt'
     sc.pp.calculate_qc_metrics(
         adata, qc_vars=["mt"], percent_top=None, log1p=False, inplace=True
     )
 
     adata = adata[
-        (adata.obs.n_genes_by_counts < 2000) & (adata.obs.pct_counts_mt < 10), :
+        (adata.obs.n_genes_by_counts <= 2000) & (adata.obs.pct_counts_mt <= 10), :
```

Comment:
Will this filter down to zero cells for small datasets?

Reply:
Yes, sometimes if a random selection of test data is bad, this possibly filter down to zero cells.

Reply:
I added this filter as above with a minimum cell number at 100.